src/VisualStudio/VisualBasic/Impl/ChangeSignature/VisualBasicChangeSignatureViewModelFactoryService.vb

```diff
@@ -41,7 +41,8 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.ChangeSignature
         End Function
 
         Public Overrides Function IsTypeNameValid(typeName As String) As Boolean
-            Return Not SyntaxFactory.ParseTypeName(typeName).ContainsDiagnostics
+            Static visualBasicParseOptions As New VisualBasicParseOptions(LanguageVersion.Latest)
```

Comment:
Probably should have a similar comment to the C# one.

Reply:
I don't think it really matters as much for the VB version. We're not adding new syntax, and VB doesn't have a concept of `Preview`.